airbyte-integrations/connectors/source-chargebee/source_chargebee/manifest.yaml

```diff
@@ -25,6 +25,7 @@ definitions:
               action: IGNORE
               error_message: "Stream is available only for Product Catalog 1.0"
         - type: DefaultErrorHandler
+          max_retries: 10
```

Comment:
My understanding was the default was 5. Why increase this to 10?

Reply:
the idea here was that due to us having a higher level of concurrency, we have a higher chance of getting rate limited. This would make the sync more resilient and when we incrementally back off we should get to a point where we can retry since the limit resets per minute.

On a tangent, I also have been thinking whether having a `max_retries` should even exist at all. Aside from the cost of extra resourcing, it feels like a sync should never crash unless the container crashed or the heartbeat times out.

Reply:
I partially agree. I think what concerns me is infinite loops and the fact that we might not be notified if there is an issue as we can't have an alert until the source ends with a code different than 0.